### PR TITLE
Update cloud-video-interop.md

### PR DESCRIPTION
**Pull request by NancyForero**
adding new section for release notes; i needed the link in the page to include it in the M365 announcement center.

### CODE REVIEW

File: Teams/cloud-video-interop.md [hunk 1]
```diff
@@ -32,6 +32,9 @@ With Microsoft Teams, you get rich online content collaboration in meetings that
 
 With Cloud Video Interop, Microsoft Teams delivers a native meeting experience for all participants – in meeting rooms or inside of Teams clients.
 
+### Release Notes 
+coming late July!
```
Comment (JohanFreelancer9) [inline Teams/cloud-video-interop.md:36]:
```suggestion
Coming late July.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by scanum — APPROVED** [review]

**Comment by scanum** [comment]
@MikePlumleyMSFT 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by MikePlumleyMSFT** [comment]
@NancyForero we don't do product announcements on the Learn site.

**Comment by NancyForero** [comment]
@MikePlumleyMSFT this is not a product announcement. 
CVI is a 3P solution for which MS makes platform releases. this is the only page we have for CVI bc of it being a 3P solution; we are solving a gap in communication for MS Teams customers that license CVI from 3P where we tell them what we from the MS have enabled for their chosen CVI provider.
we will not have another page for CVI.

**Comment by MikePlumleyMSFT** [comment]
@NancyForero when the feature is available, we can add it to the article, but we don't announce future dates in Learn articles. Please contact me internally if you have further questions.

**Pull request closed without merging**